app/main.py

```diff
@@ -16,4 +16,58 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+"""
+# parse the names of first n variables in a file
+def get_n_variable_names(file, n):
+    try:
+        with open(file, "r") as f:
+            variables = []
+            while len(variables) != n:
+                line = f.readline().split(" = ")[0]
+                if line[0].isalpha() and "import" not in line:
```

Comment:
The condition 'line[0].isalpha()' might cause an error if 'line' is an empty string. Consider checking if 'line' is not empty before accessing its first character.